test/test_conditionals.py

```diff
@@ -1,53 +1,67 @@
 import unittest2
-import translate
+import matrix
+from Unittest import Unittest
 
 
 class TestConditionals(unittest2.TestCase):
 
     def test_if(self):
-        js_code = 'if (1) {\n\tconsole.log("This is true")\n}'
-        py_code = 'if (1):\n\tprint("This is true")'
-        bash_code = 'if [[ 1 ]]; then\n\techo "This is true"\nfi'
-        java_code = 'if (1) {\n\tSystem.out.println("This is true");\n}'
-        self.assertEqual(py_code, translate.translate(js_code, 'js', 'py'))
-        self.assertEqual(js_code, translate.translate(py_code, 'py', 'js'))
-        self.assertEqual(bash_code, translate.translate(py_code, 'py', 'bash'))
-        self.assertEqual(bash_code, translate.translate(js_code, 'js', 'bash'))
-        self.assertEqual(java_code, translate.translate(js_code, 'js', 'java'))
-        self.assertEqual(py_code, translate.translate(java_code, 'java', 'py'))
+        js_code = Unittest('if (1) {\n\tconsole.log("This is true")\n}', 'js')
+        py_code = Unittest('if (1):\n\tprint("This is true")', 'py')
+        bash_code = Unittest('if [[ 1 ]]; then\n\techo "This is true"\nfi',
+                             'bash',
+                             is_input=False)
+        java_code = Unittest(
+            'if (1) {\n\tSystem.out.println("This is true");\n}', 'java')
+        matrix.matrix(self, [py_code, js_code, java_code, bash_code])
 
     def test_else(self):
-        js_code = 'if (1) {\n\tconsole.log("1 is true")\n} else {\n\tconsole.log("1 is NOT true")\n}'  # TODO: consider adding ; after console.log()
-        py_code = 'if (1):\n\tprint("1 is true")\nelse:\n\tprint("1 is NOT true")'
-        bash_code = 'if [[ 1 ]]; then\n\techo "1 is true"\nelse\n\techo "1 is NOT true"\nfi'
-        java_code = 'if (1) {\n\tSystem.out.println("1 is true");\n} else {\n\tSystem.out.println("1 is NOT true");\n}'
-        self.assertEqual(py_code, translate.translate(js_code, 'js', 'py'))
-        self.assertEqual(js_code, translate.translate(py_code, 'py', 'js'))
-        self.assertEqual(bash_code, translate.translate(py_code, 'py', 'bash'))
-        self.assertEqual(bash_code, translate.translate(js_code, 'js', 'bash'))
-        self.assertEqual(java_code, translate.translate(js_code, 'js', 'java'))
-        self.assertEqual(py_code, translate.translate(java_code, 'java', 'py'))
+        js_code = Unittest(
+            'if (1) {\n\tconsole.log("1 is true")\n} else {\n\tconsole.log("1 is NOT true")\n}',
```

Comment:
Nit: Multi-line strings would be more readable.

Reply:
The way this was broken up was done by the Google linter, so I did not want to mess with that.